dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```
@@ -259,6 +259,13 @@ jenkins:
         <%- end -%>
           slaveConnectTimeout: 100
           yamlMergeStrategy: "override"
+          volumes:
+          - emptyDirVolume:
+              memory: false
```

```suggestion
              memory: true
```

- `/tmp` in a `tmpfs` should be a sane default 
- The memory used in `/tmp` is controlled by the pod limit so it is safe